openapi/webscraping.ai/client.bal

```diff
@@ -30,109 +28,76 @@ public type ApiKeysConfig record {|
+    # + serviceUrl - URL of the target service 
+    # + return - An error if connector initialization failed 
     public isolated function init(ApiKeysConfig apiKeyConfig, http:ClientConfiguration clientConfig =  {}, string serviceUrl = "https://api.webscraping.ai") returns error? {
         http:Client httpEp = check new (serviceUrl, clientConfig);
         self.clientEp = httpEp;
-        self.apiKeys = apiKeyConfig.apiKeys.cloneReadOnly();
+        self.apiKeyConfig = apiKeyConfig.cloneReadOnly();
     }
     # Page HTML by URL
     #
-    # + url - URL of the target page
-    # + headers - HTTP headers to pass to the target page. Can be specified either via a nested query parameter (...&headers[One]=value1&headers=[Another]=value2) or as a JSON encoded object (...&headers={"One": "value1", "Another": "value2"})
-    # + timeout - Maximum processing time in ms. Increase it in case of timeout errors (5000 by default, maximum is 30000)
-    # + js - Execute on-page JavaScript using a headless browser (true by default)
-    # + proxy - Type of proxy, use residential proxies if your site restricts traffic from datacenters (datacenter by default). Note that residential proxy requests are more expensive than datacenter, see the pricing page for details.
-    # + return - Parameters validation error
-    remote isolated function getHTML(string url, record {}? headers = (), int timeout = 5000, boolean js = true, string proxy = "datacenter") returns string|error {
+    # + url - URL of the target page 
+    # + headers - HTTP headers to pass to the target page. Can be specified either via a nested query parameter (...&headers[One]=value1&headers=[Another]=value2) or as a JSON encoded object (...&headers={"One": "value1", "Another": "value2"}) 
+    # + timeout - Maximum processing time in ms. Increase it in case of timeout errors (5000 by default, maximum is 30000) 
+    # + js - Execute on-page JavaScript using a headless browser (true by default) 
+    # + proxy - Type of proxy, use residential proxies if your site restricts traffic from datacenters (datacenter by default). Note that residential proxy requests are more expensive than datacenter, see the pricing page for details. 
+    # + return - Parameters validation error 
+    remote isolated function getHTML(string url, record {}? headers = (), int timeout = 5000, boolean js = true, string proxy = "datacenter") returns Error|error {
         string  path = string `/html`;
-        map<anydata> queryParam = {"url": url, "headers": headers, "timeout": timeout, "js": js, "proxy": proxy, "api_key": self.apiKeys["api_key"]};
+        map<anydata> queryParam = {"url": url, "headers": headers, "timeout": timeout, "js": js, "proxy": proxy, "api_key": self.apiKeyConfig.apiKey};
         path = path + check getPathForQueryParam(queryParam);
-        string response = check self.clientEp-> get(path, targetType = string);
+        Error response = check self.clientEp-> get(path, targetType = Error);
```

Comment:
Every response is mapped to the `Error` record which is invalid according to the OpenAPI definition file. It seems there is an error in the code generation.

Reply:
Found the issue. At the connector generation 1st response has been taken as the target type. 
```
      responses:
        403:
          $ref: '#/components/responses/403'
        200:
          description: Success
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Account"
              example:
                remaining_api_calls: 200000
                resets_at: 1617073667
                remaining_concurrency: 100```

Please re-arrange the response order keeping the success response at the top